framework/Optimizers/GradientBasedOptimizer.py

```diff
@@ -487,9 +467,9 @@ def _getJobsByID(self,traj):
     """
     solutionUpdateList = []
     solutionIndeces = []
-    # get all the even-valued results (these are the multiple evaluations of the opt point)
+    # get all the opt point results (these are the multiple evaluations of the opt point)
     for i in range(self.gradDict['numIterForAve']):
-      identifier = i*2
```

Comment:
Just to clarify, what's the new prefix convention (the old one was odds for neighbors, evens for opt points)?

Reply:
[0 to numbAvgEvalautions] -> opt points
[numbAvgEvaluations to NumbAvgEvaluations*NumberOfNeededPerturbation] -> neighbors

Reply:
(I put a note at line 647)

Reply:
Awesome, I like this better than what was there before.  Thanks.

Reply:
👯‍♂️ 👯 🇮🇹